pages/editor.vue

```diff
@@ -147,14 +168,35 @@
         </div>
     </div>
     <div class="d-lg-none">
-        <OverlayModal :title="selectedCategory.name" :show="catalogShow" @close="catalogShow = false">
+        <OverlayModal :full="true" :show="catalogShow" @close="catalogShow = false">
+            <div class="category__description">
+                <div class="edit-icon icon text-primary">
+                    {{ iconConfig.get(selectedCategory.name) || 'default_icon' }}
+                </div>
+                <div class="navigation__title h1">{{ selectedCategory.name }}</div>
+            </div>
             <CatalogLayout
                 :selectedCategoryAssets="catalogElements"
                 :selectedCategory="selectedCategory"
                 @catalog-changed="(e) => fetchCatalogElements(e.category, e.subCategory, e.filter)"
             />
         </OverlayModal>
     </div>
+    <OverlayModal :show="goingBackPopUpShow" :full="false" @close="goingBackPopUpShow = false">
+        <DecisionPopUp
+            imgSrc="http://localhost:3000/catalog/Characters/single/Barista%20pouring4.png?raw=true"
+            title="Poof, Your hard work disappears"
+            body="Are you sure you want to delete your draft? All the changes you've made will be discarded."
+            :buttons="[
+                { name: 'Save Draft', function: 'save' },
+                { name: 'Discard all changes', function: 'discard' },
+                { name: 'Cancel', function: 'cancel' },
```

Comment:
as I said before, maybe directly pass callback function would make the use of DecisionPopUp even more generic